cuebot/src/main/java/com/imageworks/spcue/dao/postgres/DispatcherDaoJdbc.java

```diff
@@ -429,5 +431,18 @@ public boolean higherPriorityJobExists(JobDetail baseJob, VirtualProc proc) {
 
         return result;
     }
+
+    @Override
+    public boolean isTestMode() {
+        return testMode;
+    }
+
+    @Override
+    public void setTestMode(boolean enabled) {
+        testMode = enabled;
+        if (enabled) {
+            bookableShows.clear();
+        }
+    }
```

Comment:
I understand the idea of using `setTestMode` to turn off the cache, but I'm not seeing the point of keeping a state using `testMode` if `isTestMode` is never used. 

Besides that, I don't see `setTestMode` being used anywhere. Would the idea be that this method should be called on the test setup module?

Reply:
I've renamed this method to "clearCache", removed "testMode" attribute.
This method is called this way - CoreUnitDispatcherTests.setTestMode() -> Dispatcher.setTestMode(true) -> DispatchSupport.clearCache() -> DispatcherDao.clearCache().